app/views/costumes/index.html.erb

```diff
@@ -1,29 +1,36 @@
-<div class="container">
-  <div class="text-center">
-    <h1>Costumes For You</h1>
-  </div>
-  <div class="cards-index">
-    <% @costumes.each do |costume| %>
-      <div class="card-index">
-        <div class="row">
-          <div class="col-12">
-            <%= link_to costume_path(costume) do %>
-              <ul>
-                NAME
-                <li> <%= costume.name %> </li>
-                CATEGORY
-                <li> <%= costume.category %> </li>
-                DESCRIPTION
-                <li> <%= costume.description %> </li>
-                <li> PRICE $<%= costume.price %> </li>
-              </ul>
-            </div>
-          <% end %>
-        </div>
+<div class="d-flex">
+<h1 class="alert-danger text-center mx-5 mt-5 mb-3 p-1" role="alert"><strong>Costume</strong></h1>
+</div>
+
+<div class="cards-index mx-5 mt-3">
+ <% @costumes.each do |costume| %>
+  <div class="card-product" style="width: 800px;">
+    <img src="https://i.etsystatic.com/12047134/r/il/3279fc/1964607794/il_794xN.1964607794_c4i2.jpg" />
```

Comment:
We're going to need to change this to `<%= cl_image_tag costume.photo.key, width: 350, height: 200, gravity: :face, crop: :fill %>` for our second user journey (the create costume). So their cards display here with the images they upload. To do this you're going to need to pull origin master and make that .env file in the app/ folder with the cloudinary key in it, I think.